Lombiq.Analyzers.PowerShell/Rules/Measure-AutomaticVariableAlias/Measure-AutomaticVariableAlias.psm1

```diff
@@ -0,0 +1,69 @@
+﻿<#
+.SYNOPSIS
+    Replaces usages of the alias of the automatic variable ($_) with its original form ($PSItem).
```

Comment:
Why does it use the Measure verb when it replaces? Could be [Repair](https://learn.microsoft.com/en-us/powershell/scripting/developer/cmdlet/approved-verbs-for-windows-powershell-commands?view=powershell-7.3#diagnostic-verbs) for example.

Reply:
I followed the convention in https://github.com/PowerShell/PSScriptAnalyzer/blob/master/Tests/Engine/CommunityAnalyzerRules/CommunityAnalyzerRules.psm1

Repair would indicate that it fixes the violations, but that only happens if you execute the analysis with the fix switch.

Reply:
Then the synopsis and description are misleading. Based on the sample you've linked the synopsis should be something like `Uses $PSItem instead of $_.` and the description should be the justification, so about the same as the "Message" below.

Reply:
I see and I agree! How about changing the synopsis to
`Detects the usages of the alias ($_) of the automatic variable ($PSItem) and suggests to correct them.`
and the description to
`The full name of the automatic variable ($PSItem) should be used instead of its alias ($_) for consistency.`
?

Reply:
Sure.

Reply:
The point is that removing those, and their contents still leave a complete sentence, and they just add extra information.

Reply:
While, grammatically, that might be true, I'm among those developers who wouldn't know what **the automatic variable** actually is without having `$PSItem` right behind it.

For me, the parentheses add reading blockers - they pause my flow.

Reply:
TBH, I think that's fine, and they are supposed pause the flow.

But! I just realized that $PSItem is not *the* automatic variable, it's *an* automatic variable (it is *the* automatic pipeline variable), but the only one with an alias.

With that in mind, the parentheses are indeed unnecessary, so I'll fix that in dev when everything's merged, because I had to deal with a bunch of submodule updates and merge conflicts I didn't cause already.

Reply:
Now that I think about it, there are many [automatic variables](https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_automatic_variables?view=powershell-7.3) besides `$PSItem` so specifying that this is only about `$PSItem` is not just extra information.